README.md

```diff
@@ -39,9 +39,10 @@ npm run start
 ### Generate documents using fixtures
 For testing purpose, documents can be rendered without _react_ frontend, using bare `npm` command:
 ```
-npm run gen_documents [DOCUMENT_TYPE]
+npm run gen_documents [DOCUMENT_TYPE] [DECISION_TYPE]
 ```
-inserting `SPECJALNE`, `INDYWIDUALNE`, `INDYWIDUALNE_ROCZNE`, `REWALIDACYJNE`, `OPINIA` in place of `[DOCUMENT_TYPE]`
+inserting `DECISION`, `PROTOKOL`, `ZARZADZENIE`, `ZAWIADOMIENIE` in place of `[DOCUMENT_TYPE]`
```

Comment:
What do you say on  `DECYZJA` instead of `DECISION`? We will be consistent (one language).

Reply:
You're right, but... This name is a part of created document name eg: `Jan Kowalski - DECISION - 12.12.2012`. And I think it'd be more familiar for user to have that document type in polish. Currently it'd be easier for testing, because tester doesn't have to think what what does `decree` means. I suggest to add some mapping here later.